scss/samples/_colors.scss

```diff
@@ -92,6 +92,28 @@ $demo-colors: (
   'brand-pink': $brand-pink,
 );
 
+/// ## CSS custom properties
```

Comment:
> I wonder if we want to include that Herman will display the css custom property name like var(--callout) in this example. Is that by default as well?

Hm, yeah -- I'm not sure there's an easy way for us to convert the css custom property into other color formats at compile time. The only thing I can think of is running a script at runtime to get the computed `background-color` value from the swatch. So by default it will only ever show the property name, not a color value. 🤷

Reply:
@jgerigmeyer I don't think it is a dealbreaker to not show color value, but just a bit of a helpful tidbit of info since some colors are pretty similar. 

I'm thinking we can re-word the description of what we show as we say name (accent, in the above example) and then the color in hsl/a, hex, rgb/a... which in the variable example we only show color name and variable name.

Reply:
@stacyk Added a note in https://github.com/oddbird/sassdoc-theme-herman/pull/383/commits/0433eb634c297e8506558941354f74db2ae1201d